doc/developers/maintainer.rst

```diff
@@ -69,10 +102,28 @@ For more information see https://github.com/scikit-learn/scikit-learn/wiki/How-t
 
        $ twine upload dist/
 
-6. Push the documentation to the website. Circle CI should do this
-   automatically for master and <N>.<N>.X branches.
-
-7. FOR FINAL RELEASE: Update the release date in What's New
+6. For major/minor (not bug-fix release), update the symlink for ``stable``
+   in https://github.com/scikit-learn/scikit-learn.github.io::
+
+       $ cd /tmp
+       $ git clone --depth 1 --no-checkout https://github.com/scikit-learn/scikit-learn.github.io
```

Comment:
shouldn't we default on `git-ssh` really?

Reply:
Why? By far the majority of the time is spent in file transfer in which I would think the overhead is negligible in both cases...?

Reply:
Or do you mean to make push easy, assuming keys are set up? I don't find pushing with 2FA to https a big problem, personally...